pkgs/top-level/python-packages.nix

```
@@ -8647,6 +8647,8 @@ in {
 
   websockify = callPackage ../development/python-modules/websockify { };
 
+  webssh = disabledIf (!isPy3k) (callPackage ../development/python-modules/webssh { });
```

```suggestion
  webssh = callPackage ../development/python-modules/webssh { };
```

Python 2 packages are now in a separate file.

---

Isn't it that Python 2 only packages are in a separate file? With this change, I no longer get the following output:

```shell-session
$ nix build -f default.nix pythonPackages.webssh
error: webssh not supported for interpreter python2.7
```

But instead:

```shell-session
$ nix build -f default.nix pythonPackages.webssh
error: builder for '/nix/store/g9d25vi8rl1xbyrnzndh45v77jpyic8a-python2.7-webssh-1.5.3.drv' failed with exit code 1;
       last 10 log lines:
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       > *** No channel.
       > DEBUG:paramiko.transport:EOF in transport thread
       > DEBUG:paramiko.transport:EOF in transport thread
       For full logs, run 'nix log /nix/store/g9d25vi8rl1xbyrnzndh45v77jpyic8a-python2.7-webssh-1.5.3.drv'
```